packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js

```diff
@@ -189,6 +189,22 @@ describe('validation', function() {
 
     });
 
+
+    describe('icons', function() {
+
+      testTemplate('icon');
+
+
+      testTemplate('icon-raw-svg');
```

Comment:
Why do we also support this? Any particular reason?

Reply:
Raw SVG is a security concern; I#d like to not open that box.

Reply:
Images can just fine be embedded into SVG: https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Other_content_in_SVG#embedding_raster_images.

Reply:
We do not support this, [the test case provides errors](https://github.com/camunda/element-templates-json-schema/blob/50-icon-schema/packages/zeebe-element-templates-json-schema/test/fixtures/icon-raw-svg.js#L13).

Reply:
Proof of works (IMG in SVG):

![foo](https://user-images.githubusercontent.com/58601/160609748-cc18d1cc-aa32-43d0-bac3-058a9979aa5b.svg)

Reply:
> We don't support this

Aaah, great. Maybe the test cases could be renamed for claritiy, i.e. "valid-xxx", "invalid-xxx".

Reply:
Yes, I know. Still, raw SVG shouldn't be supported (what we are testing right now with this test), but only data URLs or `http(s):`

Reply:
> Maybe the test cases could be renamed for claritiy, i.e. "valid-xxx", "invalid-xxx".
will do 👍